qds_sdk/commands.py

```diff
@@ -235,28 +231,8 @@ def get_results(self, fp=sys.stdout, inline=True, delim=None, fetch=True, qlog=N
         else:
             if fetch:
                 storage_credentials = conn.get(Account.credentials_rest_entity_path)
-                boto_conn = boto.connect_s3(aws_access_key_id=storage_credentials['storage_access_key'],
-                                            aws_secret_access_key=storage_credentials['storage_secret_key'],
-                                            security_token = storage_credentials['session_token'])
-
-                log.info("Starting download from result locations: [%s]" % ",".join(r['result_location']))
-                #fetch latest value of num_result_dir
-                num_result_dir = Command.find(self.id).num_result_dir
-
-                for s3_path in r['result_location']:
-
-                    # If column/header names are not able to fetch then use include header as true
-                    if include_header.lower() == "true" and qlog is not None:
-                        write_headers(qlog, fp)
-
-                    # In Python 3,
-                    # If the delim is None, fp should be in binary mode because
-                    # boto expects it to be.
-                    # If the delim is not None, then both text and binary modes
-                    # work.
-
-                    _download_to_local(boto_conn, s3_path, fp, num_result_dir, delim=delim,
-                                       skip_data_avail_check=isinstance(self, PrestoCommand))
+                cloud = Qubole.get_cloud()
```

Comment:
Nit: Can you change the comments of get_results() to say object store instead of s3.